rio-server/send-data.js

```diff
@@ -3,7 +3,9 @@ const dgram = require('dgram');
 const config = require('./config');
 const uuid = require('node-uuid');
 const consoleOutput = require('./outputs/console-output');
-const piOutput = require('./outputs/pi-output');
+if (config.sendToPi) {
```

Comment:
Yeah without it the server always tried to start the python script which crashes the node process unless you're on a RPi